tests/data/relative_jsonpointer.json

```diff
@@ -217,5 +217,44 @@
                 "result": "awkwardly/named~variable"
             }
         ]
+    },
+    {
+        "data": [
+            "foo",
+            "bar",
+            {"baz": "value"}
+        ],
+        "tests": [
+            {
+                "start": "/0",
+                "ref": "0+1",
+                "result": "bar"
+            },
+            {
+                "start": "/1",
+                "ref": "0-1#",
+                "result": 0
+            },
+            {
+                "start": "/0",
+                "ref": "0+2/baz",
+                "result": "value"
+            },
+            {
+                "start": "/2/baz",
+                "ref": "1-2",
+                "result": "foo"
+            },
+            {
+                "start": "/2/baz",
+                "ref": "1+5",
```

Comment:
I'd suggest changing this to catch the edge case: 
`"ref": "1+1"`

Reply:
Nice catch (here and in the code).  As a former QA automation engineer, I'm a touch embarrassed to have overlooked it! 😅 Just proves the rule that devs miss things in our own code.